### DISCUSSION TITLE
New block is not getting created after 16910 blocks???????????????

### DISCUSSION THREAD

**Discussion by ghost** [ConsenSys/quorum#1598] [Q&A]
Hi All,

 My network is running since last three day and approx more than 16000 blocks were created but after that no any new block is getting created and network is up and running properly. So how can I resolve this issue?

Logs are :
-----------------------------------------------------------------------------------------------------
INFO [01-02|11:11:59.712] Writing clean trie cache to disk         path=/data/geth/triecache threads=1
INFO [01-02|11:11:59.712] Regenerated local transaction journal    transactions=0 accounts=0
INFO [01-02|11:11:59.713] Persisted the clean trie cache           path=/data/geth/triecache elapsed=1.376ms
INFO [01-02|11:30:00.715] Looking for peers                        peercount=1 tried=6 static=8
INFO [01-02|11:30:15.715] Looking for peers                        peercount=1 tried=1 static=8
INFO [01-02|11:30:28.727] Got interrupt, shutting down... 
INFO [01-02|11:30:28.728] HTTP server stopped                      endpoint=[::]:8545
INFO [01-02|11:30:28.730] Looking for peers                        peercount=0 tried=5 static=8
INFO [01-02|11:30:28.730] HTTP server stopped                      endpoint=[::]:8546
INFO [01-02|11:30:28.730] IPC endpoint closed                      url=/data/geth.ipc
INFO [01-02|11:30:28.730] Stopping all plugins                     count=0
INFO [01-02|11:30:28.730] extension service: stopped 
INFO [01-02|11:30:28.730] Ethereum protocol stopped 
INFO [01-02|11:30:28.730] Transaction pool stopped 
INFO [01-02|11:30:28.730] BFT: deactivate 
INFO [01-02|11:30:28.731] QBFT: stopping...                        address=0x401BDFcd9221CD790E7cd66EeCE303eD4D4B53B1
INFO [01-02|11:30:28.731] QBFT: stopped                            address=0x401BDFcd9221CD790E7cd66EeCE303eD4D4B53B1
INFO [01-02|11:30:28.731] Writing cached state to disk             block=16911 hash="4d5c98â€¦376d71" root="0f03f8â€¦f7dc73"
INFO [01-02|11:30:28.731] Persisted trie from memory database      nodes=14 size=2.28KiB time="158.301Âµs"  gcnodes=42 gcsize=7.86KiB gctime=9.568161ms livenodes=1 livesize=-246.00B
INFO [01-02|11:30:28.731] Writing cached state to disk             block=16910 hash="1ed6fdâ€¦92519e" root="0f03f8â€¦f7dc73"
INFO [01-02|11:30:28.731] Persisted trie from memory database      nodes=0  size=0.00B   time="1.4Âµs"      gcnodes=0  gcsize=0.00B   gctime=0s         livenodes=1 livesize=-246.00B
INFO [01-02|11:30:28.731] Writing cached state to disk             block=16784 hash="dff2c5â€¦8a34be" root="0f03f8â€¦f7dc73"
INFO [01-02|11:30:28.731] Persisted trie from memory database      nodes=0  size=0.00B   time="1.2Âµs"      gcnodes=0  gcsize=0.00B   gctime=0s         livenodes=1 livesize=-246.00B
INFO [01-02|11:30:28.731] Writing snapshot state to disk           root="d7d9afâ€¦f2e575"
INFO [01-02|11:30:28.731] Persisted trie from memory database      nodes=0  size=0.00B   time="1.2Âµs"      gcnodes=0  gcsize=0.00B   gctime=0s         livenodes=1 livesize=-246.00B
INFO [01-02|11:30:28.732] Writing clean trie cache to disk         path=/data/geth/triecache threads=4
INFO [01-02|11:30:28.735] Persisted the clean trie cache           path=/data/geth/triecache elapsed=2.449ms
INFO [01-02|11:30:28.735] Blockchain stopped 
-------------------------------------------------------------------------------------------------------------------------------------------

**Comment by baptiste-b-pegasys** [comment]
Can you provide details on the environment? Did the runtime received a kill signal?
